### DISCUSSION TITLE
Adding Maps to quasar

### DISCUSSION THREAD

**Discussion by bobwatcherx** [quasarframework/quasar#15032] [General - Components / Directives / etc]
I have a problem adding maps I'm confused in vue 3 normally you can add app.use() but in quasar , how do you add app.use()

vue3-openlayer code
```
import { createApp } from 'vue'
import App from './App.vue'

import OpenLayersMap from 'vue3-openlayers'
import 'vue3-openlayers/dist/vue3-openlayers.css'

const app = createApp(App);
app.use(OpenLayersMap)

app.mount('#app')
```
where should I set app.use if in quasar or is there a map plugin for quasar?

**Comment by MilosPaunovic** [comment]
Take a look at this example of boot file, you should follow it for your use case: https://quasar.dev/quasar-cli-webpack/boot-files#vue-i18n
